### ISSUE TITLE
Are there execute analysis only piece of code? 

### ISSUE THREAD

**Issue by joasgarcia** [CodeNarc/CodeNarc#155]
For example execute analysis only git commit changes (not for all file or for all directories)?

**Comment by chrismair** [comment]
No, there is no such facility. CodeNarc uses the Groovy compiler, so whatever is used as input would have to be valid Groovy code and be compile-able. But perhaps it would be possible to assemble the files that have changed as an input set.

**Comment by joasgarcia** [comment]
I got it.
So if I have list of changed files I can for example run analysis for only that list of files using ```-includes``` option, alright?
See this example of pre commit hook that runs [Checkstyle](http://checkstyle.sourceforge.net/) code analysis on changed files: https://gist.github.com/gkzhong/3916852

**Comment by chrismair** [comment]
Yes, if you are running CodeNarc using the command-line runner, then use  **-includes** to specify the changed files. If you are using the Ant Task then you can accomplish it using a `<fileset><include>`.

**Comment by jlstrater** [comment]
Hi! Is there an issue that needs to be resolved here or can this issue be closed?
